src/helpers/graphqlHelpers.ts

```diff
@@ -4,9 +4,10 @@ import gql from "graphql-tag"
 export type OperationType = "query" | "mutation" | "subscription" | "persisted"
 
 export interface IGraphqlRequestBody {
+  id: string
   query: string
-  variables?: object
-  extensions?: object
+  variables?: Record<string, unknown>
```

Comment:
I changed the types to Record<string, unknown> to enable the narrowing on the in operator